src/components/Modal/SearchListModal.js

```diff
@@ -1,29 +1,36 @@
 import React from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
-function SearchListModal({ datas, searchText }) {
-  const history = useHistory();
-
-  const result = datas.filter(post => {
-    return post.title.includes(searchText);
-  });
-
+function SearchListModal({ data }) {
   return (
     <section>
       <div>
         <ListContainer>
-          {result.map((post, idx) => {
-            if (idx < 5)
-              return (
-                <Link to={`/search?query=${post.title}`}>
-                  <List>
-                    <img alt="postingImg" src={post.img} />
-                    <p>{post.title}</p>
-                  </List>
-                </Link>
-              );
-          })}
+          {data &&
+            data.companies.map(company => (
+              <Link to={`/search?query=${company.name}`}>
+                <List>
+                  <p>{company.name}</p>
+                </List>
+              </Link>
+            ))}
```

Comment:
```suggestion
          {data?.companies.map(company => (
              <Link to={`/search?query=${company.name}`}>
                <List>
                  <p>{company.name}</p>
                </List>
              </Link>
            ))}
```
옵셔널 체이닝이라는 문법이 있습니다 확인해보세요!

Reply:
오... 갓연욱💪 ...

이거 엄청 맣이 쓸것 같네요